### PR TITLE
[Logs] BigQuery info for logpush

### PR DESCRIPTION
**Pull request by crwaters16**

### CODE REVIEW

File: products/logs/src/content/logpush/bigquery/index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+order: 61
+---
+
+# Enable BigQuery
+
+Configure Logpush to send batches of Cloudflare logs to BigQuery.
+
+BigQuery supports loading up to 1,500 jobs per table per day (including failures) with up to 10 million files in each load. That means you can load into BigQuery once per minute and include up to 10 million files in a load. See BigQuery's quotas for load jobs for more information.  
+
+Logpush delivers batches of logs as soon as possible, which means you could receive more than one batch of files per minute. Ensure your BigQuery job is configured to ingest files on a given time interval, like every minute, as opposed to as files are received. Ingesting files into BigQuery as each Logpush file is received could exhaust your BigQuery quota quickly.
```
Comment (kodster28) [inline products/logs/src/content/logpush/bigquery/index.md]: Suggest changing to `as opposed to when files are received`

**Review by kodster28 — APPROVED** [review]

**Comment by kodster28** [comment]
Approved from an editing perspective

**Comment by crwaters16** [comment]
@shagamemnon / @soheiokamoto , any concerns with pushing this content to dev docs now that https://github.com/cloudflare/cloudflare-gcp/pull/70 is merged in?

**Review by soheiokamoto — APPROVED** [review]

**Comment by soheiokamoto** [comment]
Sounds good! Thanks!

**Pull request merged**